aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -212,67 +213,106 @@ Tensor ConvertToChannelsLast3dTensor(const Tensor& src) {
 
 template <int kSpatialDim = 2>
 CAFFE2_API torch::class_<ConvPackedParamsBase<kSpatialDim>> register_conv_params() {
-  using SerializationType = std::tuple<
-    at::Tensor,
-    c10::optional<at::Tensor>,
-    // these are meant to be torch::List<int64_t> but
-    // it's not supported by onnx, so we'll use Tensor as
-    // a workaround
-    torch::List<at::Tensor>,
-    torch::List<at::Tensor>,
-    torch::List<at::Tensor>,
-    at::Tensor>;
   static auto register_conv_params =
     torch::class_<ConvPackedParamsBase<kSpatialDim>>(
         "quantized", "Conv" + c10::to_string(kSpatialDim) + "dPackedParamsBase")
     .def_pickle(
         [](const c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>>& params)
-        -> SerializationType { // __getstate__
+        -> ConvPackedParamsSerializationType { // __getstate__
           at::Tensor weight;
           c10::optional<at::Tensor> bias;
           std::tie(weight, bias) = params->unpack();
-          torch::List<at::Tensor> stride;
-          torch::List<at::Tensor> padding;
-          torch::List<at::Tensor> dilation;
-          at::Tensor groups;
+          std::vector<int64_t> scalar_elements;
```

Comment:
what is the format we agreed upon in the end? Looks like MS is not going to support int64_t in ONNX, so we might need to keep Tensor for scalar elements

Reply:
Given that we might want to pack all the parameters into one entry to keep it compact. maybe we can pack stride/padding/dilation/transpose into one Tensor.

Reply:
I a mwriting a new serialization with more "scalable" and "reusable" serialization type. Will send an update within an hour

Reply:
So I see that RNN uses vector of double/int -- can we use that? Reference: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/RNN.cpp#L59-L64